### DISCUSSION TITLE
document Architectural Decision Records

### DISCUSSION THREAD

**Discussion by KevinMGranger** [dora-metrics/pelorus#810] [Ideas]
There are some choices we made in the past where we debated, weighed alternatives and tradeoffs, but didn't document our choices or why we chose them.

The idea of [Architectural Decision Records](https://www.redhat.com/architect/architecture-decision-records) was meant for this.

We should add them to our [architecture docs](https://github.com/konveyor/pelorus/pull/753).

Related: #727

**Comment by mpryc** [comment]
Converting to discussion.

**Comment by mateusoliveira43** [comment]
Closing as this was implemented and we have it documented https://pelorus.readthedocs.io/en/latest/Development/#architectural-decision-records
